bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/api/ControllerVariables.java

```diff
@@ -40,6 +40,11 @@ public class ControllerVariables {
     private ControllerVariables() {
     }
 
+    public static ControllerVariables empty() {
+        ControllerVariables info = new ControllerVariables();
+        return info;
+    }
```

Comment:
Is it used ?
I have the feeling it is not.

Reply:
Ah, indeed, it is not :) I'll remove it